CHANGELOG.md

```diff
@@ -51,6 +51,8 @@ Main (unreleased)
 
 - Update OracleDB Exporter dependency to 0.5.0 (@schmikei)
 
+- Update azure-metrics-exporter to latest (@kgeckhart)
```

Comment:
Can you put an explicit version here? "latest" will change with time.

Reply:
Updated, unfortunately the upstream author hasn't done a release since Sept 2022